project.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
```

Comment:
You really got the project right as required in freecodecamp, I would suggest that you use spaces instead of tab indentation to tally with HTML semantics. Good job